resources/source/tutorials/tutorial-words/en.json

```diff
@@ -0,0 +1,78 @@
+{
+    "imagine.hello": {
+        "message": "Hello!",
+        "description": "A character saying 'Hello!'. Step 2 of Imagine a World"
+    },
+    "imagine.imagine-if": {
+        "message": "Imagine if...",
+        "description": "A character is saying this. This phrase is meant to give the audience the idea that they could write sentences like 'Imagine if people could fly' or 'Imagine if animals could talk.' Step 2 of Imagine a World"
+    },
+    "imagine.lets-explore": {
+        "message": "Let’s explore!",
+        "description": "A character is exploring a new background (in one case it's a city, in the other case it's underwater). Doesn't need to be exactly literal. Appears in Step 14 of Imagine a World and Step 8 of Talking Tales."
+    },
+    "story.welcome-magic-school": {
+        "message": "Welcome to Magic School!",
+        "description": "A wizard is saying this in a magical backdrop. Can be shortened to 'Welcome to school' or just 'Welcome' if needed. Step 4 of Create a Story"
+    },
+    "story.going-on-quest": {
+        "message": "I’m going on a quest!",
+        "description": "A student character is telling their professor that they are going to go on a quest. Step 7 of Create a Story"
+    },
+    "fly.time-to-fly": {
+        "message": "Time to fly!",
+        "description": "A character says this at the beginning of the Make it Fly activity. Doesn't need to be literal; just needs to express that the character is going to fly. Step 4 of Make it Fly."
+    },
+    "animations-that-talk.lets-go": {
+        "message": "Let’s go!",
+        "description": "A character is inviting the audience to come with them. Step 5 of Create Animations That Talk and Step 9 of Code a Cartoon."
+    },
+    "animations-that-talki-am-a-robot": {
+        "message": "I am a robot",
+        "description": "This is the first line of a 3-line song that a robot sings. Doesn't need to be literal; just needs to be playful. Step 8 of Create Animations That Talk."
+    },
+    "animations-that-talki-like-to-dance-a-lot": {
+        "message": "I like to dance a lot",
+        "description": "This is the second line of a 3-line song that a robot sings. Doesn't need to be literal; just needs to be playful. Step 8 of Create Animations That Talk."
+    },
+    "animations-that-talkbeep-boop-bop": {
+        "message": "Beep boop bop",
+        "description": "This is the third line of a 3-line song that a robot sings. It is onomatopoeia. Doesn't need to be literal at all; just needs to be playful. Step 8 of Create Animations That Talk."
```

Comment:
missing `.` Also, I wouldn't use onomatopoeia in the description as I don't expect every translator to know what that means. Instead I would say that it is the third line that the robot sings, and it is nonsense words that are supposed to sound like the noises a robot makes.